### PR TITLE
Implemented void SetupAccessibility () for FindInFilesDIalog

### PR DESCRIPTION
**Pull request by Jammer77**

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs [hunk 1]
```diff
@@ -235,8 +236,37 @@ public static string FormatPatternToSelectionOption (string pattern, bool regex)
 				UpdateSensitivity ();
 				return true;
 			});
+
+			//SetupAccessibility ();
```
Comment (mrward) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: This line seems to be commented out.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs [hunk 2]
```diff
@@ -235,8 +236,37 @@ public static string FormatPatternToSelectionOption (string pattern, bool regex)
...
+
 		}
-		
+
+		private void SetupAccessibility ()
+		{
+			comboboxentryFind.SetCommonAccessibilityAttributes ("FindInFilesDialog.comboboxentryFind",
+												"Find",
+												GettextCatalog.GetString ("Enter string for find"));
```
Comment (mrward) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: I would probably word this as "Enter string to find"

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs [hunk 3]
```diff
@@ -235,8 +236,37 @@ public static string FormatPatternToSelectionOption (string pattern, bool regex)
...
+
+			searchentryFileMask.SetCommonAccessibilityAttributes ("FindInFilesDialog.searchentryFileMask",
+												"File Mask",
+												GettextCatalog.GetString ("Enter the file mask"));
+			searchentryFileMask.SetAccessibilityLabelRelationship (labelFileMask);
+
+
+			comboboxentryReplace.SetCommonAccessibilityAttributes ("FindInFilesDialog.comboboxentryReplace",
+												"Replace",
+												GettextCatalog.GetString ("Enter string for replace"));
```
Comment (mrward) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: I would probably use: "Enter string to replace".

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs [hunk 4]
```diff
@@ -235,8 +236,49 @@ public static string FormatPatternToSelectionOption (string pattern, bool regex)
 				UpdateSensitivity ();
 				return true;
 			});
+
+			SetupAccessibility ();
+
 		}
-		
+
+		private void SetupAccessibility ()
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: Please remove the `private` modifier on this and the methods below

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs (hunk 1 above)
Comment (Jammer77) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: I did it in next commit.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs (hunk 4 above)
Comment (Jammer77) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: Done

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs (hunk 2 above)
Comment (Jammer77) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: Did it!

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs [hunk 5]
```diff
@@ -235,8 +236,49 @@ public static string FormatPatternToSelectionOption (string pattern, bool regex)
...
 		}
-		
+
+		void SetupAccessibility ()
+		{
+			comboboxentryFind.SetCommonAccessibilityAttributes ("FindInFilesDialog.comboboxentryFind",
+												"Find",
+												GettextCatalog.GetString ("Enter string to find"));
+			comboboxentryFind.SetAccessibilityLabelRelationship (labelFind);
+
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: There are 4 unnecessary white lines here.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs [hunk 6]
```diff
@@ -235,8 +236,49 @@ public static string FormatPatternToSelectionOption (string pattern, bool regex)
...
+
+
+
+			searchentryFileMask.SetCommonAccessibilityAttributes ("FindInFilesDialog.searchentryFileMask",
+												"File Mask",
+												GettextCatalog.GetString ("Enter the file mask"));
+			searchentryFileMask.SetAccessibilityLabelRelationship (labelFileMask);
+
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: Remove unnecessary white line.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs [hunk 7]
```diff
@@ -235,8 +236,49 @@ public static string FormatPatternToSelectionOption (string pattern, bool regex)
...
+		}
+
+		void SetupAccessibilityForReplace ()
+		{
+			comboboxentryReplace.SetCommonAccessibilityAttributes ("FindInFilesDialog.comboboxentryReplace",
+											"Replace",
+											GettextCatalog.GetString ("Enter string to replace"));
+			comboboxentryReplace.SetAccessibilityLabelRelationship (labelReplace);
+
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: Unnecessary white line.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs [hunk 8]
```diff
@@ -235,8 +236,49 @@ public static string FormatPatternToSelectionOption (string pattern, bool regex)
...
+		}
+
+
+		void SetupAccessibilityForPath ()
+		{
+			comboboxentryPath.SetCommonAccessibilityAttributes ("FindInFilesDialog.comboboxentryPath",
+												"Path",
+												GettextCatalog.GetString ("Enter the Path"));
+			comboboxentryPath.SetAccessibilityLabelRelationship (labelPath);
+
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: Same

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs (hunk 6 above)
Comment (Jammer77) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.FindInFiles/FindInFilesDialog.cs]: Lines removed.

**Pull request merged**